src/LndClient/Data/BtcEnv.hs

```diff
@@ -0,0 +1,22 @@
+module LndClient.Data.BtcEnv
+  ( BtcConfig (..),
```

Comment:
Maybe let's call it `BtcEnv` just for consistency with `LndEnv`?

Reply:
Agree, done!